liam2/alignment.py

```diff
@@ -295,11 +317,26 @@ def _display_unaligned(self, expressions, ids, columns, unaligned):
 
     def compute(self, context, score, need, filter=None, take=None, leave=None,
                 expressions=None, possible_values=None, errors='default',
-                frac_need='uniform', link=None, secondary_axis=None):
+                frac_need='uniform', link=None, secondary_axis=None,
+                method='default'):

```

Comment:
the default method should not be called "default", but "bysorting" or "sort" or something more descriptive like that